build.gradle

```diff
@@ -111,27 +108,24 @@ dependencies {
     testImplementation "org.apache.flink:flink-streaming-java_${scalaBinaryVersion}:${flinkVersion}:tests"
 
     testRuntimeOnly "com.vladsch.flexmark:flexmark-all:0.35.10"
-    testRuntime 'org.pegdown:pegdown:1.4.2'
-    testCompile "info.picocli:picocli:4.2.0"
+    testRuntimeOnly 'org.pegdown:pegdown:1.4.2'
+    testImplementation "info.picocli:picocli:4.2.0"
 }
 
 publishing {
     publications {
-        mavenJava(MavenPublication) {
-            from components.java
-            artifactId = 'hri-flink-pipeline-core'
-        }
-        mavenJava(MavenPublication) {
-            artifact jarTests
+        gpr(MavenPublication) {
+            from(components.java)
```

Comment:
I'm not sure I understand this change, could you explain?

Reply:
Yeah, I was wondering this too.  What is `gpr`?